src/main/scala/beam/sim/population/PopulationAttributes.scala

```diff
@@ -139,10 +143,37 @@ case class AttributesOfIndividual(
     valueOfTime * generalizedTime
   }
 
+  private def getSituationForVOT(
+    originActivity: Option[Activity],
+    destinationActivity: Option[Activity]
+  ): Set[SituationMultiplier] = {
+    (destinationActivity, originActivity, age) match {
+      case (Some(origin), Some(destination), Some(travelerAge)) =>
```

Comment:
We still get a new val `origin` from `destinationActivity`. Is this right?

Reply:
good catch, adjusted